Week-3/3.1/Memorise.js

```diff
@@ -0,0 +1,30 @@
+function memorise(fn) {
+    const cache = new Map();
+    return function (...args) {
+        const key = args.toString();
+        if (cache.has(key)) {
+            return cache.get(key);
+        }
+
+        cache.set(key, fn(...args));
+        return cache.get(key);
+
```

Comment:
please some use some indentation plugin or format your code before raising the PR